fdbserver/MoveKeys.actor.cpp

```diff
@@ -2434,15 +2504,24 @@ ACTOR Future<Void> cleanUpDataMove(Database occ,
 				Optional<Value> val = wait(tr.get(dataMoveKeyFor(dataMoveId)));
 				if (val.present()) {
 					dataMove = decodeDataMoveValue(val.get());
-					ASSERT(!dataMove.ranges.empty());
-					TraceEvent(SevVerbose, "CleanUpDataMoveMetaData", dataMoveId)
+					if (dataMove.ranges.empty()) {
```

Comment:
When DD init, will it call `cleanUpDataMoveCore`  for a data move marked as 'Deleting`? Shall we throw `need_background_datamove_cleanup` here?